### PR TITLE
feat(listbox-button): added support for description element

### PR DESCRIPTION
**Pull request by agliga**
## Description
* Added support for pulling value separately. So its not breaking I made the original case work in case value is not found
* Updated docs

__NOTE__ Styles wont work properly until skin is published.

### CODE REVIEW

File: docs/makeup-listbox-button/index.html [hunk 1]
```diff
@@ -324,7 +324,60 @@ <h3>Unselected, Floating label</h3>
             </div>
           </div>
         </div>
-        
+
+        <h3>Unselected - with descripton</h3>
```
Comment (ianmcburnie) [inline docs/makeup-listbox-button/index.html:328]: Can we add an example to the Listbox page also, please.

File: docs/makeup-listbox-button/index.html [hunk 2]
```diff
@@ -324,7 +324,60 @@ <h3>Unselected, Floating label</h3>
...
+              <span class="btn__text">Color: -</span>
+              <svg class="icon icon--chevron-down-12" focusable="false" height="10" width="14" aria-hidden="true">
+                <use xlink:href="../icons.svg#icon-chevron-down-12"></use>
+              </svg>
+            </span>
+          </button>
+          <div class="listbox-button__listbox" hidden>
+            <div class="listbox-button__options" role="listbox">
+              <div class="listbox-button__option" role="option">
+                <span class="listbox-button__value">Red</span>
```
Comment (ianmcburnie) [inline docs/makeup-listbox-button/index.html:348]: Let's also add the clipped punctuation on these examples.

Comment (agliga) [reply docs/makeup-listbox-button/index.html:348]: Done! Good catch

File: docs/makeup-listbox-button/index.html (hunk 1 above)
Comment (agliga) [reply docs/makeup-listbox-button/index.html:328]: I was initially considering to add it, but there was no visual difference, which is why I did not add it.
I did add it now just for consistency.

**Review by ianmcburnie — APPROVED** [review]

**Pull request merged**